.github/workflows/ci-test.yml

```diff
@@ -57,4 +57,9 @@ jobs:
 
       - name: Build with Hugo ${{ matrix.hugo }}
         working-directory: exampleSite
-        run: HUGO_THEME="Binario" hugo --themesDir ../.. --i18n-warnings -v
+        run: |
+          function ver { printf "%03d%03d%03d" $(echo "$1" | tr '.' ' '); }
+          HUGO_VERSION=$(hugo version | grep -Eo '[0-9]\.[0-9]+\.[0-9]+')
```

Comment:
I'm using `hugo version` here and not `${{ matrix.hugo }}` because I would like to know what's the version of 'latest', and not use the 'latest' string.